swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.item import Item
+
+class Clothing (Item):
+    def __init__ (self, condition = 0, category = "Clothing"):
```

Comment:
Style: There shouldn't be a space between the name and arguments:

```suggestion
class Clothing(Item):
    def __init__(self, condition = 0, category = "Clothing"):
```